skflow/estimators/base.py

```diff
@@ -170,11 +168,12 @@ def fit(self, X, y, logdir=None):
             self._trainer.initialize(self._session)
             self._initialized = True
             # Sets up summary writer for later optional visualization
```

Comment:
This comment is for the next block (remove one indentation). Just curious, could the logic below be more simpler for the next if statement? e.g. initialize it to None and then simply check logdir and whether it's None? Maybe I missed the logic somewhere else.